nessie/sql_templates/create_sis_advising_notes_schema.template.sql

```diff
@@ -277,6 +277,22 @@ CREATE SCHEMA {redshift_schema_sis_advising_notes_internal};
 -- Internal tables
 --------------------------------------------------------------------
 
+CREATE OR REPLACE FUNCTION {redshift_schema_sis_advising_notes_internal}.to_utc_iso_string(date_string VARCHAR)
+RETURNS VARCHAR
+STABLE
+AS $$
+  from datetime import datetime
+  import pytz
+
+  d = datetime.strptime(date_string, '%d-%b-%y %I.%M.%S.%f000 %p')
+  d = pytz.timezone('America/Los_Angeles').localize(d)
+  return d.astimezone(pytz.utc).isoformat()
+$$ language plpythonu;
+
+GRANT EXECUTE
+ON function {redshift_schema_sis_advising_notes_internal}.to_utc_iso_string(VARCHAR)
+TO GROUP {redshift_app_boa_user}_group;
+
 CREATE TABLE {redshift_schema_sis_advising_notes_internal}.advising_notes
```

Comment:
The use of "STABLE" above is explained at https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_FUNCTION.html